source/stylesheets/card.scss

```diff
@@ -6,7 +6,30 @@
   border-radius: $base-border-radius;
   height: $card-height;
   margin: $base-vh-spacing auto 0;
+  transition: $base-transition;
+  position: absolute;
+  left: 50%;
+  margin-left: -$card-width / 1.77;
```

Comment:
Can you accomplish the same thing with only the `left` value, and without needing the margin?

Reply:
I actually followed this guide to make sure the spacing works: https://robots.thoughtbot.com/positioning

left:50% positions it relative to the left edge, not via center of the card, so it needs a bit of margin to move it exactly to the center.